crates/ruff/resources/test/fixtures/flake8_pyi/PYI011.pyi

```diff
@@ -13,7 +13,7 @@ def f11(*, x: str = "x") -> None: ...  # OK
 def f13(
     x: list[
         str
-    ] = [  # Error PYI011 Only simple default values allowed for typed arguments
+    ] = [  # OK
```

Comment:
It looks like nested containers are disallowed -- is that right? And do we have any tests for those?

Reply:
- Yes, I will add the added tests from PyCQA/flake8-pyi#358.